common/models/user.js

```diff
@@ -869,9 +1003,21 @@ module.exports = function(User) {
     );
 
     UserModel.remoteMethod(
+      'verify',
+      {
+        description: 'Trigger user\'s identity verification.',
+        accepts: [
+          {arg: 'uid', type: 'string', required: true, http: {source: 'form'}},
+          {arg: 'options', type: 'object', http: 'optionsFromRequest'},
+        ],
+        http: {verb: 'post', path: '/verify'},
+      }
+    );
+
+    UserModel.remoteMethod(
       'confirm',
       {
-        description: 'Confirm a user registration with email verification token.',
+        description: 'Confirm a user registration with identity verification token.',
```

Comment:
This change does not make sense to me. Isn't this method invoked when the user clicks on the link sent in the verification email? In which case it is an _email verification token_ IMO.